stats/stats/src/data_source/types.rs

```diff
@@ -39,6 +41,7 @@ impl<'a> UpdateParameters<'a> {
     pub fn query_parameters(
         db: &'a DatabaseConnection,
         is_multichain_mode: bool,
+        multichain_filter: Option<Vec<u64>>,
```

Comment:
kinda nit and not obvious:
`query_parameter` is for parameters passed when querying data from stats service. Since both "update charts" and "query data from stats" use these params, I've created this method to not overcomplicate querying part when only update-specific stuff is changed.

So, since we can't filter by chain id when querying data available at `stats` service, I suggest to remove this parameter and pass `None` to `Self {..}`

I will also appreciate documenting the info above in the code, since it caused a confusion! (if you have time)

Reply:
also the same seems to be true for `multichain_mode`. plus it's always `false` when calling `query_parameters` (like `multichain_filter` is `None`)